src/components/Header/Header.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+type Props = {
+  activeTodosLength: number,
+};
+
+export const Header: React.FC<Props> = ({ activeTodosLength }) => (
+  <header className="todoapp__header">
+    {/* this buttons is active only if there are some active todos */}
```

Comment:
code says the same - comment is redundant (there is no complication logic that should be described in comments)
fix the same everywhere